.github/workflows/formatting.yml

```diff
@@ -0,0 +1,33 @@
+---
+name: 'Verify formatting and Makefile.am sort order'
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+    name: verify formatting and Makefile.am sort order
+    # on a ubuntu-20.04 VM
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          fetch-depth: 5
+          submodules: recursive
+      - run: |
+          sudo apt-get update
```

Comment:
In the build log, this step now calls itself `sudo apt-get update`, which looks like the install bit is missing. Putting them on one line with `&&` works, making them two steps also works, and giving the step a `name:` is cleanest.

Reply:
(The next two steps also look confusing in the build log for the same reason.)